mapboxcli/scripts/directions.py

```diff
@@ -7,15 +7,17 @@
 
 @click.command(short_help="Routing between waypoints.")
 @click.argument('waypoints', nargs=-1)
-@click.option('--profile', default="mapbox.driving", type=click.Choice([
-              'mapbox.driving', 'mapbox.walking', 'mapbox.cycling']),
-              help="Mapbox direction profile id")
+@click.option('--profile', default="mapbox.driving",
+              help="Mapbox direction profile id, must be one of {0}".format(
+                  mapbox.Directions().valid_profiles))
```

Comment:
How about making this a `click.Choice` populated from `mapbox.Directions().valid_profiles`?